crates/evm_mlir/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[workspace]
+members = ["bench/revm_comparison"]
+
+[package]
+name = "evm_mlir"
+version = "0.1.0"
+edition = "2021"
+repository = "https://github.com/lambdaclass/evm_mlir/"
```

Comment:
outdated

Reply:
Updated file in [d3d0947](https://github.com/lambdaclass/ethereum_rust/pull/548/commits/d3d0947b30963a3a95463c67471cfb1cfe072a04)